Android/tun2socks/build.gradle

```diff
@@ -1,2 +1,77 @@
-configurations.maybeCreate("default")
-artifacts.add("default", file('tun2socks.aar'))
\ No newline at end of file
+// We use Android library plugin to get the Android SDK path.
+plugins {
+    id('com.android.library')
+}
+
+android {
+    compileSdkVersion 33
```

Comment:
define it as a reusable global variable, so we can reference it in both tun2socks/build.gradle and app/build.gradle